app/src/androidTest/java/app/foodfinderapp/RestaurentUnitTest.kt

```diff
@@ -14,7 +14,7 @@ import org.junit.Assert.*
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 @RunWith(AndroidJUnit4::class)
-class ContextUnitTest {
+class RestaurentUnitTest {
```

Comment:
better to name test classes according to what is being tested in the class than something that is general so i made the name more descriptive